mod/gradle.properties

```diff
@@ -1,14 +1,15 @@
 org.gradle.jvmargs=-Xmx2048M
 
-maven_group=gjum.minecraft.mapsync
 archives_base_name=map-sync
-
+maven_group=gjum.minecraft.mapsync
 mod_version=2.0.1-1.18.2
 
 minecraft_version=1.18.2
+enabled_platforms=fabric,forge
+architectury_version=4.9.83
 
 # https://fabricmc.net/versions.html
-fabric_loader_version=0.13.3
-fabric_api_version=0.51.1+1.18.2
+fabric_loader_version=0.14.9
+fabric_api_version=0.58.0+1.18.2
```

Comment:
We're not bundling any of these, so why update the dependency? Wouldn't this only mean that we now exclude some users who have to rely on an older version?

Reply:
i think its fine personally

Reply:
tbh, the syntax highlighting wasn't working so I had no idea these values weren't used... I can remove the values entirely if they're useless, if you'd prefer